### PR TITLE
refactor(new python version): Bump python to version 3

### PR DESCRIPTION
**Pull request by kaaquist**
Let's bump it up! New version of Python to version 3

### CODE REVIEW

File: Dataflow/pipeline/local_helpers/cm_helper.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 
 def init_cm(credentials_path):
-  """Initializes the Campaign Manager API.
```
Comment (oslaru) [inline Dataflow/pipeline/local_helpers/cm_helper.py:31]: line 17 and 19 not needed. Errors in googleapiclient imports line 22 and 23, may need to install this library through requirements.txt

**Review by oslaru — CHANGES_REQUESTED** [review]

File: Dataflow/pipeline/requirements.txt [hunk 2]
```diff
@@ -1,40 +1,2 @@
...
-pytz==2019.1
-pyyaml==3.13
-requests==2.22.0
-rsa==4.0
-six==1.12.0
-typing==3.6.6
-urllib3==1.25.3
-google-auth-httplib2==0.0.3
+apache-beam[gcp]==2.34.0
+google-cloud-videointelligence==1.9.0
```
[the remark this answers is not in the document]
Comment (kaaquist) [reply Dataflow/pipeline/requirements.txt:2]: So if you run the `pip install -r requirements.txt`. it should install a bunch of libs that are required for apache-beam. 
Here is the list: 
```sh
Collecting apache-beam[gcp]==2.34.0
  Using cached apache_beam-2.34.0-cp38-cp38-manylinux2010_x86_64.whl (11.7 MB)
Collecting google-cloud-videointelligence==1.9.0
  Using cached google_cloud_videointelligence-1.9.0-py2.py3-none-any.whl (189 kB)
Collecting numpy<1.21.0,>=1.14.3
  Using cached numpy-1.20.3-cp38-cp38-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (15.4 MB)
Collecting pytz>=2018.3
  Using cached pytz-2021.3-py2.py3-none-any.whl (503 kB)
Processing /home/kasper/.cache/pip/wheels/8e/70/28/3d6ccd6e315f65f245da085482a2e1c7d14b90b30f239e2cf4/future-0.18.2-py3-none-any.whl
Processing /home/kasper/.cache/pip/wheels/07/35/78/e9004fa30578734db7f10e7a211605f3f0778d2bdde38a239d/dill-0.3.1.1-py3-none-any.whl
Collecting oauth2client<5,>=2.0.1
  Using cached oauth2client-4.1.3-py2.py3-none-any.whl (98 kB)
Collecting requests<3.0.0,>=2.24.0
  Using cached requests-2.26.0-py2.py3-none-any.whl (62 kB)
Collecting protobuf<4,>=3.12.2
  Using cached protobuf-3.19.1-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.1 MB)
Collecting grpcio<2,>=1.29.0
  Using cached grpcio-1.42.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (4.0 MB)
Collecting pyarrow<6.0.0,>=0.15.1
  Using cached pyarrow-5.0.0-cp38-cp38-manylinux2014_x86_64.whl (23.6 MB)
Collecting hdfs<3.0.0,>=2.1.0
  Using cached hdfs-2.6.0-py3-none-any.whl (33 kB)
Collecting python-dateutil<3,>=2.8.0
  Using cached python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
Processing /home/kasper/.cache/pip/wheels/a5/f2/87/b7c4b9d5915716d94e8bf2e2f3bfbbd73bb5fe2a98677a59cb/avro_python3-1.9.2.1-py3-none-any.whl
Processing /home/kasper/.cache/pip/wheels/ca/5a/02/f3acf982a026f3319fb3e798a8dca2d48fafee7761788562e9/crcmod-1.7-cp38-cp38-linux_x86_64.whl
Collecting httplib2<0.20.0,>=0.8
  Using cached httplib2-0.19.1-py3-none-any.whl (95 kB)
Collecting typing-extensions<4,>=3.7.0
  Using cached typing_extensions-3.10.0.2-py3-none-any.whl (26 kB)
Collecting orjson<4.0; python_version >= "3.6"
  Using cached orjson-3.6.4-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (250 kB)
Collecting pydot<2,>=1.2.0
  Using cached pydot-1.4.2-py2.py3-none-any.whl (21 kB)
Collecting fastavro<2,>=0.21.4
  Using cached fastavro-1.4.7-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (2.6 MB)
Collecting pymongo<4.0.0,>=3.8.0
  Using cached pymongo-3.12.1-cp38-cp38-manylinux2014_x86_64.whl (546 kB)
Collecting google-cloud-spanner<2,>=1.13.0; extra == "gcp"
  Using cached google_cloud_spanner-1.19.1-py2.py3-none-any.whl (255 kB)
Collecting google-cloud-datastore<2,>=1.8.0; extra == "gcp"
  Using cached google_cloud_datastore-1.15.3-py2.py3-none-any.whl (134 kB)
Collecting google-cloud-recommendations-ai<=0.2.0,>=0.1.0; extra == "gcp"
  Using cached google_cloud_recommendations_ai-0.2.0-py2.py3-none-any.whl (180 kB)
Collecting google-cloud-dlp<2,>=0.12.0; extra == "gcp"
  Using cached google_cloud_dlp-1.0.0-py2.py3-none-any.whl (169 kB)
Collecting google-cloud-language<2,>=1.3.0; extra == "gcp"
  Using cached google_cloud_language-1.3.0-py2.py3-none-any.whl (83 kB)
Processing /home/kasper/.cache/pip/wheels/d7/54/79/85de1824f2f4175fb4960c72afb10045d86700c3941dc73685/google_apitools-0.5.31-py3-none-any.whl
Collecting google-cloud-bigquery<3,>=1.6.0; extra == "gcp"
  Using cached google_cloud_bigquery-2.30.1-py2.py3-none-any.whl (203 kB)
Collecting grpcio-gcp<1,>=0.2.2; extra == "gcp"
  Using cached grpcio_gcp-0.2.2-py2.py3-none-any.whl (9.4 kB)
Collecting google-cloud-bigquery-storage>=2.6.3; extra == "gcp"
  Using cached google_cloud_bigquery_storage-2.10.1-py2.py3-none-any.whl (171 kB)
Collecting google-cloud-pubsub<2,>=0.39.0; extra == "gcp"
  Using cached google_cloud_pubsub-1.7.0-py2.py3-none-any.whl (144 kB)
Collecting google-cloud-core<2,>=0.28.1; extra == "gcp"
  Using cached google_cloud_core-1.7.2-py2.py3-none-any.whl (28 kB)
Collecting google-auth<3,>=1.18.0; extra == "gcp"
  Using cached google_auth-2.3.3-py2.py3-none-any.whl (155 kB)
Collecting google-cloud-bigtable<2,>=0.31.1; extra == "gcp"
  Using cached google_cloud_bigtable-1.7.0-py2.py3-none-any.whl (267 kB)
Collecting google-cloud-vision<2,>=0.38.0; extra == "gcp"
  Using cached google_cloud_vision-1.0.0-py2.py3-none-any.whl (435 kB)
Collecting cachetools<5,>=3.1.0; extra == "gcp"
  Using cached cachetools-4.2.4-py3-none-any.whl (10 kB)
Collecting google-api-core[grpc]<2.0.0dev,>=1.6.0
  Using cached google_api_core-1.31.4-py2.py3-none-any.whl (93 kB)
Collecting pyasn1>=0.1.7
  Using cached pyasn1-0.4.8-py2.py3-none-any.whl (77 kB)
Collecting rsa>=3.1.4
  Using cached rsa-4.7.2-py3-none-any.whl (34 kB)
Collecting six>=1.6.1
  Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting pyasn1-modules>=0.0.5
  Using cached pyasn1_modules-0.2.8-py2.py3-none-any.whl (155 kB)
Collecting charset-normalizer~=2.0.0; python_version >= "3"
  Using cached charset_normalizer-2.0.7-py3-none-any.whl (38 kB)
Collecting urllib3<1.27,>=1.21.1
  Using cached urllib3-1.26.7-py2.py3-none-any.whl (138 kB)
Collecting idna<4,>=2.5; python_version >= "3"
  Using cached idna-3.3-py3-none-any.whl (61 kB)
Collecting certifi>=2017.4.17
  Using cached certifi-2021.10.8-py2.py3-none-any.whl (149 kB)
Processing /home/kasper/.cache/pip/wheels/56/ea/58/ead137b087d9e326852a851351d1debf4ada529b6ac0ec4e8c/docopt-0.6.2-py2.py3-none-any.whl
Collecting pyparsing<3,>=2.4.2
  Using cached pyparsing-2.4.7-py2.py3-none-any.whl (67 kB)
Processing /home/kasper/.cache/pip/wheels/8f/b9/13/fce3d62261f63c01b28281fe6a9d704a7af65d96ff2c88552e/grpc_google_iam_v1-0.12.3-py3-none-any.whl
Collecting proto-plus>=1.15.0
  Using cached proto_plus-1.19.8-py3-none-any.whl (45 kB)
Collecting fasteners>=0.14
  Using cached fasteners-0.16.3-py2.py3-none-any.whl (28 kB)
Collecting packaging>=14.3
  Using cached packaging-21.3-py3-none-any.whl (40 kB)
Collecting google-resumable-media<3.0dev,>=0.6.0
  Using cached google_resumable_media-2.1.0-py2.py3-none-any.whl (75 kB)
Collecting libcst>=0.2.5
  Downloading libcst-0.3.22-py3-none-any.whl (517 kB)
     |████████████████████████████████| 517 kB 1.1 MB/s 
Requirement already satisfied: setuptools>=40.3.0 in /home/kasper/developer/virtualenv/oculi-dev-test/lib/python3.8/site-packages (from google-auth<3,>=1.18.0; extra == "gcp"->apache-beam[gcp]==2.34.0->-r Dataflow/pipeline/requirements.txt (line 1)) (44.0.0)
Collecting googleapis-common-protos<2.0dev,>=1.6.0
  Using cached googleapis_common_protos-1.53.0-py2.py3-none-any.whl (198 kB)
Collecting google-crc32c<2.0dev,>=1.0
  Using cached google_crc32c-1.3.0-cp38-cp38-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (37 kB)
Collecting typing-inspect>=0.4.0
  Using cached typing_inspect-0.7.1-py3-none-any.whl (8.4 kB)
Collecting pyyaml>=5.2
  Using cached PyYAML-6.0-cp38-cp38-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_12_x86_64.manylinux2010_x86_64.whl (701 kB)
Collecting mypy-extensions>=0.3.0
  Using cached mypy_extensions-0.4.3-py2.py3-none-any.whl (4.5 kB)
ERROR: google-api-core 1.31.4 has requirement google-auth<2.0dev,>=1.25.0, but you'll have google-auth 2.3.3 which is incompatible.
ERROR: google-cloud-core 1.7.2 has requirement google-auth<2.0dev,>=1.24.0, but you'll have google-auth 2.3.3 which is incompatible.
Installing collected packages: numpy, pytz, future, dill, pyasn1, rsa, pyparsing, httplib2, six, pyasn1-modules, oauth2client, charset-normalizer, urllib3, idna, certifi, requests, protobuf, grpcio, pyarrow, docopt, hdfs, python-dateutil, avro-python3, crcmod, typing-extensions, orjson, pydot, fastavro, pymongo, cachetools, google-auth, packaging, googleapis-common-protos, google-api-core, google-cloud-core, grpc-google-iam-v1, google-cloud-spanner, google-cloud-datastore, proto-plus, google-cloud-recommendations-ai, google-cloud-dlp, google-cloud-videointelligence, google-cloud-language, fasteners, google-apitools, google-crc32c, google-resumable-media, google-cloud-bigquery, grpcio-gcp, mypy-extensions, typing-inspect, pyyaml, libcst, google-cloud-bigquery-storage, google-cloud-pubsub, google-cloud-bigtable, google-cloud-vision, apache-beam
```

[the remark this answers is not in the document]
Comment (oslaru) [reply Dataflow/pipeline/requirements.txt]: tried with setup_environment.sh, that runs 
# Install requirements
echo "Installing requirements with pip..."
sleep 2
pip install -r pipeline/requirements.txt

but maybe I had to wait to the end of the indexing process...

[the remark this answers is not in the document]
Comment (kaaquist) [reply Dataflow/pipeline/requirements.txt]: I can see I'm missing `google-cloud-storage` and `google-api-python-client`. I have added these. I do think we at some point could optimize on this here, but for now I'm just installing everything :grimacing:

**Review by oslaru — APPROVED** [review]
LGTM

**Pull request merged**